### PR TITLE
[2.4] meson: Prioritize native WolfSSL package over built-in SSL library if it has the correct configuration

### PR DESCRIPTION
**Pull request by dgsga**

### CODE REVIEW

**Comment by dgsga** [comment]
@rdmark, only Debian and Ubuntu WolfSSL packages appear to have the correct config...

File: meson.build [hunk 1]
```diff
@@ -582,7 +613,7 @@ elif crypto.found()
 else
     have_ssl = false
     warning(
-        'Either LibreSSL or OpenSSL version 1.1 is required for DHX and RANDNUM support',
+        'Built-in SSL, LibreSSL or OpenSSL version 1.1 is required for DHX and RANDNUM support',
```
Comment (rdmark) [inline meson.build:616]: Did you mean to say "Built-in WolfSSL"?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -91,7 +92,8 @@ jobs:
             openrc \
             openssl-dev \
             pkgconfig \
-            rpcsvc-proto-dev
+            rpcsvc-proto-dev \
+            wolfssl-dev
```
Comment (rdmark) [inline .github/workflows/build.yml:96]: Isn't it misleading to install the wolfssl-dev for this OS if it won't be usable by netatalk?

Comment (rdmark) [reply .github/workflows/build.yml:96]: (Same comment for other OSes except Debian and Ubuntu below)

Comment (dgsga) [reply .github/workflows/build.yml:96]: I thought it was a good way to test the logic of the macro. Will remove now I know it works

File: meson.build (hunk 1 above)
Comment (dgsga) [reply meson.build]: Fixed

**Review by rdmark — APPROVED** [review]

**Pull request merged**